flask_jsonapi/resource_repositories.py

```diff
@@ -22,34 +22,69 @@ def update(self, id, **data):
         raise exceptions.NotImplementedMethod('Updating is not implemented')
 
 
+class ResourceRepositoryViewMixin:
```

Comment:
Move classes as commit "Refactor".

Reply:
It's a part of this commit as it require class reordering to make it work, I think.

Reply:
Sorry, order is important.
+2 :+1: